cli/src/main/kotlin/commands/UploadResultToPostgresCommand.kt

```diff
@@ -86,7 +86,17 @@ class UploadResultToPostgresCommand : CliktCommand(
         val ortResult = readOrtResult(ortFile)
 
         val postgresConfig = globalOptionsForSubcommands.config.scanner.storages?.values
-            ?.filterIsInstance<PostgresStorageConfiguration>()?.singleOrNull()
+            ?.filterIsInstance<PostgresStorageConfiguration>()?.let { configs ->
+                if (configs.size > 1) {
+                    val config = configs.first()
```

Comment:
I feel a bit uneasy with this change. Selecting the first storage configuration seems rather arbitrary. The use case to have multiple database connections configured in the scanner options does not seem to be really supported currently. So, isn't the current behavior, to fail if there is ambiguity, safer than using a connection that might not be intended?

For a long-term solution, it may be necessary to refactor the way external connections are declared in `ort.conf`; e.g. have a separate section for this and allow assigning them a name. This would probably make the configuration clearer. I remember, in the past we had used SW360 for different purposes, and the connection had to be defined at multiple places.

Reply:
I agree that it's not a nice solution because it's arbitrary but I need a quick fix for this without refactoring the configuration. After all, if the first connection is not the one a user wants they could always create a separate configuration file for the command.